server/norace_test.go

```diff
@@ -3746,10 +3746,15 @@ func TestNoRaceJetStreamClusterMaxConsumersAndDirect(t *testing.T) {
 // Make sure when we try to hard reset a stream state in a cluster that we also re-create the consumers.
 func TestNoRaceJetStreamClusterStreamReset(t *testing.T) {
 	// Speed up raft
+	omin, omax, ohb := minElectionTimeout, maxElectionTimeout, hbInterval
 	minElectionTimeout = 250 * time.Millisecond
 	maxElectionTimeout = time.Second
 	hbInterval = 50 * time.Millisecond
-	defer setDefaultRaftTimeouts()
+	defer func() {
```

Comment:
Fine like that. If we end-up having to do that more often in the future (or change more than those 3), then we will make setting the raft values a function that will be invoked in the test init() and called as a defer here. But for now, this is ok.

Reply:
Ah gotcha.